### ISSUE TITLE
Migrate to python 3...

### ISSUE THREAD

**Issue by drvinceknight** [drvinceknight/Computing_for_mathematics#17]
- [ ] Find out tickables that are not compatible (potentially doc test lab sheets...)
- [ ] Find out videos that are not python 3
- [ ] Redo.

**Comment by Huaraz2** [comment]
@drvinceknight 
```
print ("Good luck with that")
```

**Comment by drvinceknight** [comment]
Hehe :+1:  lol :sob:

**Comment by alcarney** [comment]
Would you consider adding type hints, or do you think that would confuse things since they are not necessary for the code to actually run?

**Comment by drvinceknight** [comment]
> Would you consider adding type hints, or do you think that would confuse things since they are not necessary for the code to actually run?

Yes and no. It would be a huge revamp to include this everywhere but I think I might make one or two new videos where I could go in to this (more than the one thing about types that is already there). Any suggestions (PRs are very welcome here by the way)?

**Comment by chelyabinsk** [comment]
I went through all tickables and solutions up to week 6, and made them run in python 3. (Mainly just adding brackets to print) as well as solutions for tests, and handouts.
Pretty much all code works fine, only thing that has changed is when you want create a list from 0-100 you would have to say that it is a list. l = list(range(101)).
Code in videos also works, as long as you add brackets.
Yea..

**Comment by drvinceknight** [comment]
Thanks @chelyabinsk, I'm going to have to change the videos etc... Some would find that confusing. Hopefully I'll do it this Summer! :)

**Comment by chelyabinsk** [comment]
Also, first 2 screenshots for the 2nd week should be updated as well

**Comment by drvinceknight** [comment]
:+1:
